avaframe/com1DFA/checkCfg.py

```diff
@@ -65,7 +65,7 @@ def checkCellSizeKernelRadius(cfg):
     return cfg
 
 
-def checkCfgFrictionModel(cfg, relVolume=''):
+def checkCfgFrictionModel(cfg, inputSimFiles, relVolume=''):
```

Comment:
Update all checkCfgFrictionModel calls with new parameter (there are still some in test_checkCfg)